tensorflow_probability/python/distributions/ordered_logistic_test.py

```diff
@@ -0,0 +1,156 @@
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# Dependency imports
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+
+from tensorflow_probability.python.internal import test_util
+
+tfd = tfp.distributions
+
+
+@test_util.test_all_tf_execution_regimes
+class OrderedLogisticTest(test_util.TestCase):
+
+  def setUp(self):
+    self._rng = np.random.RandomState(123)
+    super(OrderedLogisticTest, self).setUp()
+
+  def testBatchShapes(self):
+
+    for test in ["cutpoints", "location", "both"]:
+
+      for batch_shape in ([], [1], [1, 2, 3]):
+
+        if test == "cutpoints":
+          cutpoints = tf.broadcast_to(
+              tf.constant([1., 2.], dtype=tf.float64), batch_shape + [2])
+          location = tf.constant(0., dtype=tf.float64)
+        elif test == "location":
+          cutpoints = tf.constant([1., 2.], dtype=tf.float64)
+          location = self._rng.randn(*batch_shape)
+        elif test == "both":
+          cutpoints = tf.broadcast_to(
+              tf.constant([1., 2.], dtype=tf.float64), batch_shape + [2])
+          location = self._rng.randn(*batch_shape)
+
+        dist = tfd.OrderedLogistic(cutpoints, location)
+
+        self.assertAllEqual(dist.batch_shape, batch_shape)
+        self.assertAllEqual(
+            self.evaluate(dist.batch_shape_tensor()), batch_shape)
+
+        self.assertAllEqual(dist.event_shape, [])
+        self.assertAllEqual(self.evaluate(dist.event_shape_tensor()), [])
+
+        log_probs_shape = tf.shape(dist.categorical_log_probs())
+        self.assertAllEqual(self.evaluate(log_probs_shape), batch_shape + [3])
+
+        sample_shape = tf.shape(dist.sample(seed=test_util.test_seed()))
+        self.assertAllEqual(self.evaluate(sample_shape), batch_shape)
+
+        sample_shape_n = tf.shape(
+            dist.sample([4, 5], seed=test_util.test_seed()))
+        self.assertAllEqual(self.evaluate(sample_shape_n), [4, 5] + batch_shape)
+
+  def testProbs(self):
+
+    # survival functions
+    # P(Y > 0) = sigmoid(1) = 0.7310586
+    # P(Y > 1) = sigmoid(0) = 0.5
+    # P(Y > 2) = sigmoid(-1) = 0.26894143
+
+    # probs
+    # P(Y = 0) = 1. - sigmoid(1) = 0.2689414
+    # P(Y = 1) = sigmoid(1) - sigmoid(0) = 0.2310586
+    # P(Y = 2) = sigmoid(0) - sigmoid(-1) = 0.23105857
+    # P(Y = 3) = sigmoid(-1) = 0.26894143
+    expected_probs = [0.2689414, 0.2310586, 0.23105857, 0.26894143]
+    dist = tfd.OrderedLogistic([-1., 0., 1.], 0.)
```

Comment:
Since this is a 2 parameter distribution, prefer putting the kwargs in
```suggestion
    dist = tfd.OrderedLogistic(cutpoints=[-1., 0., 1.], location=0.)
```

Reply:
hopefully fixed in https://github.com/tensorflow/probability/pull/753/commits/aa6ea95066499cb3c65b4f211abc9e38ac0eaa0f